google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/ConnectionWorker.java

```diff
@@ -720,7 +722,7 @@ private AppendRequestAndResponse pollInflightRequestQueue() {
   }
 
   /** Thread-safe getter of updated TableSchema */
-  public synchronized TableSchema getUpdatedSchema() {
+  public synchronized TableSchemaAndTimestamp getUpdatedSchema() {
```

Comment:
This is breaking change. Lets just add a new method instead of change this old method.

Reply:
This method should not been used as public,
let's fallback to package private

Reply:
Yeah, I agree that for ConnectionWorker this method shouldn't be public at all. But it seems the method on StreamWriter also changed?